kubernetes/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+***Changed***:
+
+* Update the Kubernetes Overview dashboard to the new responsive-grid layout ([#16013](https://github.com/DataDog/integrations-core/pull/16013))
```

Comment:
I think that we don't need a changelog for dashboards.

Reply:
That's true, the changelog only contains things that are shipped with the agent

Reply:
Sure, I'll remove this 👍